src/include/btmem.h

```diff
@@ -205,8 +205,8 @@ struct __wt_ovfl_reuse {
  * this way so that overall the lookaside table is append-mostly), a counter
  * (used to ensure the update records remain in the original order), and the
  * record's key (byte-string for row-store, record number for column-store).
- * The value is the WT_UPDATE structure's transaction ID, timestamp, update's
- * prepare state, update type and value.
+ * The value is the WT_UPDATE structure's transaction ID, timestamp, durable
+ * timestamp, update's prepare state, update type and value.
```

Comment:
This list is getting long - I think a line per item would make it more understandable.